### DISCUSSION TITLE
is there any vue.js projects ?

### DISCUSSION THREAD

**Discussion by hamidovz** [EddieHubCommunity/support#6413] [General]
i want to contribute to an open source project but i couldn't find any vue.js project here.  i can contribute projects written in express.js too. could you help me please ?

**Comment by Pradumnasaraf** [comment]
Hey, 

Currently, in EddieHub we don't have projects related to **Vue.js** and **Express**.

But, you can projects related to them here:

https://github.com/topics/vue
https://github.com/topics/express

**Comment by NaNshekhar04** [comment]
Hey @codewithbug 

Looking to contribute to **Vue** Open source project?
Check this out :-
https://github.com/cloudforet-io/console
